taxonomies/ingredients.txt

```diff
@@ -29307,6 +29307,7 @@ es:proteína de guisante hidrolizada,proteínas de guisante hidrolizadas
 <en:pea protein
 en:pea protein isolate
 es:aislado de proteina de guisante
+de:Erbsenproteinisolat
```

Comment:
ideally, this would go above the es: entry so the languages are in alphabetical order (after en:)

Reply:
Oh, yeah. Is there a simple way to change it in this pull request?

Reply:
any commits you push to the branch the pull request is from will be automatically added on to the pull request.